app/controllers/decidim/verify_wo_registration/verifications_controller.rb

```diff
@@ -58,7 +60,7 @@ def component
 
       # Should we validate that only 'direct' verification handlers are enabled?
       def validate_verification_workflow_manifests!
-        return if all_verifications_of_type_direct?(component)
+        return if Decidim::VerifyWoRegistration::ApplicationHelper.verify_wo_registration_custom_modal?(component)
```

Comment:
You've change this check by calling helper method
https://github.com/CodiTramuntana/decidim-verify_wo_registration/blob/e909116b44e5fa781d5ad260be70f472b27c815b/app/helpers/decidim/verify_wo_registration/application_helper.rb#L8
that makes double check, same as here before plus `component.settings.try(:supports_without_registration)` that's strong. Seems good for me :smile:

Reply:
yes, I added this check to disable the controller if the feature is not enabled but in fact the verification process is implemented in the `VerifyWoRegistrationForm#verify_against_enabled_authorization_handlers`.
I've updated the description of the PR